cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/ZooKeeperReconciler.java

```diff
@@ -880,25 +880,19 @@ protected Future<Void> deletePersistentClaims() {
      */
     protected Future<Void> maybeDeleteControllerZnode() {
```

Comment:
It's not a "maybe" anymore because it's going to delete the `/controller` znode so I would rename it `deleteControllerZnode`